src/main/java/LoginPage.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage {
+    private final WebDriver driver;
+    private final String loginPageURL = "https://mail.yandex.com/";
+    private final By enterButton = By.xpath("//*[@id=\"index-page-container\"]/div/div[2]/div/div/div[4]/a[2]");
```

Comment:
I will pretend that I didn't see this locator

Reply:
Locator was changed =)))